mws/resource_mws_workspaces.go

```diff
@@ -521,23 +200,15 @@ func ResourceMwsWorkspaces() common.Resource {
 			s["is_no_public_ip_enabled"].DiffSuppressFunc = func(k, old, new string, d *schema.ResourceData) bool {
 				return old != ""
 			}
+			s["is_no_public_ip_enabled"].Default = true
```

Comment:
We should handle a case of `is_no_public_ip_enabled = false`. It's marked as `omitempty`, so it won't be set in the payload